orchestrator/taste-orchestrator.py

```diff
@@ -2216,15 +2290,15 @@ def InvokeAadl2GlueCandCompile(baseDir, lock):
                 # Patch calls to _step functions, sometimes they have 0 param, sometimes they don't
                 # so look at the header files...
                 mysystem('LINES=`grep "_step.*int_T.*tid" ../../"%s"/"%s"/*h  2>/dev/null | wc -l` ; if [ $LINES -eq 1 ] ; then for i in *.c ; do cat "$i" | sed "s,_step(),_step(0)," > a_temp_name && mv a_temp_name "$i" ; done ; fi ; exit 0' % (baseDir, baseDir))
-                mysystem("\"$GNATGCC\" -c %s -I ../../auto-src %s %s %s %s %s %s %s %s %s *.c" % (
+                mysystem("\"$GNATGCC\" -c %s -I ../../auto-src %s %s %s %s %s %s %s %s %s %s *.c" % (
                     cflagsSoFar + CalculateCFLAGS(baseDir, withPOHIC=False) + CalculateUserCodeOnlyCFLAGS(baseDir),
                     bUseSimulinkMakefiles[baseDir][2],
-                    scadeIncludes, simulinkIncludes, micropythonIncludes, cIncludes, guiIncludes, adaIncludes, cyclicIncludes, rtdsIncludes))
+                    scadeIncludes, simulinkIncludes, micropythonIncludes, cIncludes, guiIncludes, adaIncludes, cyclicIncludes, rtdsIncludes, rosBridgeIncludes))
```

Comment:
`' '.join(*Includes)` allows to keep the format string constant in case this list changes again in the future.

Reply:
Can you actually use the `*` for local variables?